readme.md

```diff
@@ -136,18 +136,18 @@ Because of the way `gulp-concat` handles file paths, you may need to set `cwd` a
 
 ```js
 const gulp = require('gulp');
-const rev = require('gulp-rev');
 const sourcemaps = require('gulp-sourcemaps');
 const concat = require('gulp-concat');
 
-exports.default = () => (
-	gulp.src('src/*.js')
+exports.default = async () => {
+	const {default: rev} = await import('gulp-rev');
```

Comment:
Why dynamic import?

Reply:
➡️ https://github.com/sindresorhus/gulp-rev/pull/267/commits/1d6480d2f95fd54a3dae53ea0c05665935962b85

My mistake, I thought that we could import the other packages of the examples using ES import